addons/web_editor/static/src/js/wysiwyg/widgets/color_palette.js

```diff
@@ -327,6 +331,13 @@ export class ColorPalette extends Component {
             }
             this.state.customDefaultColor = defaultColor;
         }
+        const notSelectedSolidColor = ['theme', 'common', 'common_grays'].every(picker =>
+            !this.pickers[picker]?.querySelector('button.selected')
+        );
+
+        if (notSelectedSolidColor) {
+            this._selectTabFromButton(this.el.querySelectorAll('button')[1]);
```

Comment:
Will it always be 1 ? Did not check in details but isn't it error prone ?

Reply:
> Will it always be 1 ?

From the value of `this.el.querySelectorAll('button')` it will be always 1.

> Did not check in details but isn't it error prone?

No, it won't give any error.